backend/src/StarResults.cjs

```diff
@@ -242,7 +242,7 @@ function splitWinners(finalists, candidates, matrix) {
     return { winners: [], losers: [] };
   }
   if (candidateOrder.length === 1) {
-    return { winners: candidateOrder, losers: [] };
+    return { winners: candidateOrder, losers: [],votes};
```

Comment:
Should votes get it's own key in the dictionary?

```
return { winners: candidateOrder, losers: [], votes: votes};
```

Reply:
I'm not certain but I think without the key it just uses the variable name as the key. There are other instances later in this function where we return doesn't have any keys. This did point out that I wasn't using consistent names in the return objects in the edge cases though which might cause an issue.